libs/UIHelpers/Vec2.h

```diff
@@ -0,0 +1,16 @@
+#include "imgui/imgui.h"
+struct vec2f {
```

Comment:
Do you have any linear algebra library already in your dependencies? In my time, we often used [Eigen](https://eigen.tuxfamily.org/index.php?title=Main_Page) instead of having to define our own geometric primitives like this one... I'm surprised nothing else in the code needs linear algebra...

If not useful for anything else, probably not worth bringing it, but keep it in mind

Reply:
That's code of the transfer function library, not ours. I took it out of the main implementation because now it's shared between the histogram and the transfer function widget.